### ISSUE TITLE
Profile picture becomes distorted in member profile when browser size is decreased

### ISSUE THREAD

**Issue by RiosNicholas** [colorstackorg/oyster#137]
### Description

The user's profile picture becomes distorted (shrunken horizontally) when the display becomes smaller 3

### Steps to Reproduce

1. Go to the ColorStack Member Profile website
2. Click on 'Profile' on the sidebar
3. Decrease the width of your browser 
4. See error.

### Expected Behavior

The user's profile picture will be shrunken horizontally and become vertically off-center.

### Screenshots

<img width="1166" alt="Screenshot 2024-04-12 at 3 02 18 PM" src="https://github.com/colorstackorg/oyster/assets/85625773/8ddd147c-c4a9-4dd4-8611-83d91fc4d8b0">
<img width="743" alt="Screenshot 2024-04-12 at 3 02 08 PM" src="https://github.com/colorstackorg/oyster/assets/85625773/13e93bdf-f358-4348-8258-d69d2c31c92c">
<img width="449" alt="Screenshot 2024-04-12 at 3 01 59 PM" src="https://github.com/colorstackorg/oyster/assets/85625773/03b23e42-1515-41c7-81cc-62d1c23d1312">

### Additional Context

This occurs on the page where you edit your personal profile. This bug was detected on Firefox on my MacBook.

**Comment by JustinhSE** [comment]
@RiosNicholas this might be a MacOS issue because on my windows, my pfp does not get distorted

**Comment by ciaracade** [comment]
I can take this on :)! It would be my first open source contribution

**Comment by RiosNicholas** [comment]
I was planning to take it on, but I'm having issues getting my environment set up 😅. Please do work on it, though

**Comment by ciaracade** [comment]
I tried to replicate this on MacOS with chrome and it seems to be working fine for me. Are there any additional steps to simulating the bug?
<img width="558" alt="Screenshot 2024-04-14 at 11 50 12 PM" src="https://github.com/colorstackorg/oyster/assets/62266962/0c59b467-76a8-4ad6-a04d-a70066d36c9c">
<img width="565" alt="Screenshot 2024-04-14 at 11 50 30 PM" src="https://github.com/colorstackorg/oyster/assets/62266962/9bf52bb6-7209-4c4d-bc6d-8a1dd7cf775f">
<img width="789" alt="Screenshot 2024-04-14 at 11 50 39 PM" src="https://github.com/colorstackorg/oyster/assets/62266962/6d59c6ee-3048-4c0e-bffd-8629d25c42c2">

**Comment by RiosNicholas** [comment]
Interesting... I'm on Firefox, so I wonder if the browser is part of the problem 🤔

**Comment by ciaracade** [comment]
I tried to replicate it on Firefox and also started having issues with the build. Interesting, maybe there's something deeper? I couldn't get past the OTP screen.

**Comment by RiosNicholas** [comment]
You're also having issues setting up your environment, or what do you mean? I just checked again and am still able to replicate the error 🤔

**Comment by ramiAbdou** [comment]
@ciaracade I just assigned this ticket to you! Can't wait for you to get your first open-source contribution in!

Just a note - I think to reproduce this issue, you have to make your headline long enough to take 2 or more lines. It seems like that's causing the profile picture to shrink!

**Comment by JustinhSE** [comment]
Update for @ciaracade this issue now extends to safari users when I hit profile 😔

**Comment by RiosNicholas** [comment]
Hey @ciaracade, are you still planning to work on this? I might take a look this weekend

**Comment by RiosNicholas** [comment]
Hey @ramiAbdou, should I wait to proceed since I'm not assigned to this issue? Thanks

**Comment by ramiAbdou** [comment]
@RiosNicholas Just confirmed with Ciara that you can take this one. Reassigning to you!

**Comment by RiosNicholas** [comment]
I'm doing some investigating right now, and while I was unable to reproduce the bug in the dev environment, I found what is causing the issue. I noticed that having a long word (without spaces) as a header didn't result in word breaks. Rather, it would just proceed beyond the viewport width.
![Screenshot 2024-05-11 at 12 21 11 AM](https://github.com/colorstackorg/oyster/assets/85625773/eb82d931-1781-4dd4-94ae-b47f524217b6)

Initially, I didn't think implementing a `word-break` tailwind class into the `Text` component would be important, but I decided to do the same thing in the live member profile. I found that having long words will eventually cause the profile picture to disappear because there isn't space for the text to overflow in the deployed version's headline.

![image](https://github.com/colorstackorg/oyster/assets/85625773/50769ab1-f92b-4ef1-a2ee-e9ad709ad74e)
![Screenshot 2024-05-11 at 12 58 09 AM](https://github.com/colorstackorg/oyster/assets/85625773/310b88c9-cb9b-4155-a3c9-cd3fcd0ff83f)
![Screenshot 2024-05-11 at 12 58 40 AM](https://github.com/colorstackorg/oyster/assets/85625773/a96e3cfd-f566-4a75-9fcd-1f98d4d1091b)

This draws me to conclude that the solution may include implementing a word wrap (word-break) prop in the Text component, and I'll work on that. @ramiAbdou
